client/signup/steps/design-picker/index.jsx

```diff
@@ -7,8 +7,16 @@ import { connect } from 'react-redux';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import StepWrapper from 'calypso/signup/step-wrapper';
 import { submitSignupStep } from 'calypso/state/signup/progress/actions';
+import { getRecommendedThemes as fetchRecommendedThemes } from 'calypso/state/themes/actions';
+import { getRecommendedThemes } from 'calypso/state/themes/selectors';
 import './style.scss';
 
+// Ideally this data should come from the themes API, maybe by a tag that's applied to
+// themes? e.g. `link-in-bio` or `no-fold`
+const STATIC_PREVIEWS = [ 'bantry', 'sigler', 'miller', 'pollard', 'paxton', 'jones', 'baker' ];
+
+const SUBJECTS_TO_EXCLUDE = [ 'podcast' ];
```

Comment:
I don't know that this is necessarily the same as an Anchor theme. I believe the ones excluded from the design picker are templates that require Anchor functionality, whereas Spearhead is a (newer and popular) blog-style theme made for podcasters. Let's double-check before excluding that.

Reply:
pdgK6S-be-p2

Reply:
No special filtering, I'll show all recommended themes: pdgK6S-be-p2#comment-341